ratis-client/src/main/java/org/apache/ratis/client/api/DataStreamApi.java

```diff
@@ -37,4 +39,6 @@
 public interface DataStreamApi {
   /** Create a stream to write data. */
   DataStreamOutput stream();
+
+  DataStreamOutput stream(RaftClientRequest request);
```

Comment:
Same as RaftClient, RaftClientRequest is for internal use only in Ratis. Please don't add this. (Similar to AsyncApi, the RaftClientRequest is for internal use only.)